docs/release_notes/qa_analytics.csv

```diff
@@ -8,3 +8,4 @@ version,qa_start_time,qa_end_time,human_count,human_hours,recipes_total,recipes_
 1.1.1,2025-04-07T11:00:00Z,2025-04-08T00:00:00Z,4,6,65,65,2,AI QA engineer runs recipes post-release; new recipe tested by AI QA: Dremio data connector
 1.1.2,2025-04-14T08:00:00Z,2025-04-15T06:00:00Z,8,12,65,65,1,AI QA engineer runs recipes post-release. Dremio data connector recipe failed.
 1.2.0,2025-04-29T00:06:07Z,2025-04-29T05:23:13Z,7,6,66,66,1,AI QA engineer runs recipes post-release. Dremio data connector recipe failed.
+1.2.1,2025-05-08T04:55:00Z,2025-05-09T00:02:00Z,7,6,74,73,1,AI QA engineer runs recipes post-release. Dremio data connector recipe failed.

```

Comment:
```suggestion
1.2.1,2025-05-08T04:55:00Z,2025-05-09T00:02:00Z,7,6,66,6,1,AI QA engineer runs recipes post-release. Dremio data connector recipe failed.

```

The official count at https://spiceai.org/cookbook is what we use for this number.